services/repository/adopt.go

```diff
@@ -143,6 +139,21 @@ func adoptRepository(ctx context.Context, repoPath string, repo *repo_model.Repo
 		}
 	}
 
+	// Don't bother looking this repo in the context it won't be there
```

Comment:
The codes are duplicated with `adoptRepository` added in #30878

Reply:
Sorry, I don't understand your meanings. 🤔